inc/PluginUpgraderSkin.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
@amartya-dev instead of an override of the functions in `Plugin_Upgrader_Skin` we can make use of https://developer.wordpress.org/reference/classes/wp_ajax_upgrader_skin/ and pass that to `Plugin_Upgrader`
```
		$skin     = new \WP_Ajax_Upgrader_Skin();
		$upgrader = new \Plugin_Upgrader( $skin );\
```

Reply:
Even then we will have the problem of this running the parent feedback and that printing stuff in the API response, the parent uses `echo` which just blurps out the errors as HTML in the API response, that is a problem when parsing it as JSON.

Thus, the only reason I have even overridden this is to make sure it prints nothing and the response is clean, some of the checks about plugin existing / upgrade not available are already present on the controller level.

Other than that, the upgrader skin works fine for me

Reply:
Actually my bad, I mis read the parent, this should not print, will re-factor